nucleus-api/src/main/java/io/github/nucleuspowered/nucleus/api/service/NucleusKitService.java

```diff
@@ -21,6 +22,12 @@
     Set<String> getKitNames();
 
     /**
+     * Gets all the kits currently in NucleusPlugin.
+     * @return A {@link Map} of names to {@link Kit}s.
```

Comment:
Probably set the return type to `ImmutableMap`? You can just use static word and mention that this map does not reflect the kits dynamically.